benchmarks/benchmark_serving/benchmark_serving.py

```diff
@@ -0,0 +1,1373 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Can you add comment with a link to where this file came from and what was changed?

Reply:
done